doc/classes/EditorInspector.xml

```diff
@@ -14,6 +14,14 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="edit">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<description>
+				Shows the properties of the given [param object] in this inspector for editing.
```

Comment:
It should also mention that `null` should be passed to stop editing an object.

This is important because it can cause a crash if trying to edit an object that was freed.